docs/api/setting.md

```diff
@@ -19,7 +19,7 @@ Fetch a single `Setting`.
 
 ### PARAMS:
 
-*  **`key`** value must be a non-blank string.
+*  **`key`** string with length longer than 1
 
```

Comment:
Caused by: https://github.com/metabase/metabase/blob/bcm-defendpoint-autogen-dox-polish/src/metabase/api/setting.clj#L44

`{key su/NonBlankString}` -> `{key [:string {:min 1}]}`

Reply:
Is this accurate? I think of "longer than 1" meaning 2 or more.